src/octopeer-github/content/ContentController.ts

```diff
@@ -171,6 +183,28 @@ class ContentController {
     }
 
     /**
+     * Makes sure that START_WATCHING_PR and STOP_WATCHING_PR are logged to the database.
+     */
+    private hookFocusAndBlurToDom(database: DatabaseAdaptable) {
```

Comment:
The ordering doesn't make sense at all at the moment. But since other branches also have changes in this class, we can better wait with changing the order tomorrow after everything is merged :)